README.md

```diff
@@ -1,5 +1,6 @@
 # single-spa-playground
-[![Build Status](https://travis-ci.org/CanopyTax/single-spa-playground.svg?branch=master)](https://travis-ci.org/CanopyTax/single-spa-playground)
+
+[![Build Status](https://travis-ci.com/single-spa/single-spa-playground.svg?branch=master)](https://travis-ci.com/single-spa/single-spa-playground)
```

Comment:
This also switches us from travis-ci.org to travis-ci.com, which is now encouraged for open source projects